torchbenchmark/models/hf_GPT2_large/metadata.yaml

```diff
@@ -0,0 +1,8 @@
+eval_benchmark: false
+eval_deterministic: false
+eval_nograd: true
+train_benchmark: false
+train_deterministic: false
+not_implemented:
```

Comment:
Can you add `device: cuda` here to skip the CUDA unit test? Looks like it caused CUDA OOM.

Reply:
Thanks, I added this and the test is now passing.